x-pack/plugins/enterprise_search/public/applications/app_search/components/create_engine/constants.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+
+export const CREATE_ENGINE_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.createEngine.title',
+  {
+    defaultMessage: 'Create an Engine',
+  }
+);
+
+export const CREATE_ENGINE_FORM_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.createEngine.form.title',
+  {
+    defaultMessage: 'Name your Engine',
+  }
+);
+
+export const CREATE_ENGINE_FORM_ENGINE_NAME_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.createEngine.form.engineName.label',
+  {
+    defaultMessage: 'Engine Name',
+  }
+);
+
+export const ALLOWED_CHARS_NOTE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.createEngine.form.engineName.allowedCharactersNote',
```

Comment:
[not a change request, just me thinking out loud about i18n] is `Note` the best generic label to use for things like this? I was wondering if maybe `helpText` would be a bit more specific for this use case, but I also like `note` just fine